packages/wallets/src/features/cfd/modals/DxtradeEnterPasswordModal/DxtradeEnterPasswordModal.tsx

```diff
@@ -49,59 +69,26 @@ const DxtradeEnterPasswordModal = () => {
                 text='Create Deriv MT5 password'
             />
         );
-    };
+    }, [hide, history, isLoading, isSuccess, onSubmit, password]);
```

Comment:
can this be separated into its own file?

Reply:
This specific to this case. I think it should be here